c++/build-support/REAMD.md

```diff
@@ -0,0 +1,30 @@
+# Build support
+
+The Python scripts under the folder provide capabilities for formatting code.
+Make sure you've installed `clang-format-13`, `clang-tidy-13` and `clang-apply-replacements-13` and cmake could find them.
```

Comment:
Why not use a higher version? The latest is 19 already.

Reply:
> Why not use a higher version? The latest is 19 already.

https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md
It seems that the ubuntu22.04 docker image of GitHub doesn't support clang-19 yet. We are using 13 right now, maybe we could upgrade to 15.